driver_cpl/bld/build-namelist

```diff
@@ -176,7 +176,7 @@ EOF
 EOF
 
 # The Build::NamelistDefinition module provides utilities to validate that the output
-# namelists are consistent with the namelist definition file
+# namelists are consistent with the namelist definition files
```

Comment:
Yes - the perl version of build-namelist is still being used (I wish it were not - but its going to be a bit of a larger refactoring effort to move this to python).